flink-job/src/main/resources/log4j2.properties

```diff
@@ -1,7 +1,7 @@
-loogers=rootLooger
+rootLogger.level=INFO
+rootLogger.appenderRef.console.ref=STDOUT
+
 appender.console.type=Console
 appender.console.name=STDOUT
 appender.console.layout.type=PatternLayout
-appender.console.layout.pattern=%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n
-rootLogger.level=INFO
-rootLogger.appenderRef.console.ref=STDOUT
\ No newline at end of file
+appender.console.layout.pattern=%d{HH:mm:ss.SSS} [%t] %-5level %logger{1.} - %msg%n
```

Comment:
What will be the result of using "1." as the length parameter?

Reply:
it shortens the class name by abbreviating the package names. e.g. `o.a.f` instead of `org.apache.flink`

Reply:
Ah, nice! I am for it.